lib/ClangImporter/SwiftBridging/swift/bridging

```diff
@@ -158,6 +158,13 @@
 #define SWIFT_NONCOPYABLE \
   __attribute__((swift_attr("~Copyable")))
 
+/// Specifies the ownership semantics (passed as owned/retained or
+/// unowned/unretained) for C++ free/global functions and static methods having
+/// a FRT as return type.
```

Comment:
It's better not to use the "FRT" abbreviation in user-facing comments. Someone who is not very familiar with C++ interop wouldn't know what it stands for.

Reply:
I see, should I replace "FRT" with "SWIFT_SHARED_REFERENCE type" ?

Reply:
I would say "... methods returning types annotated as SWIFT_SHARED_REFERENCE".